can/generated_2021/uwrt_mars_rover_can_dump.txt

```diff
@@ -1,4 +1,4 @@
-================================= Messages =================================
+[?1049h[22;0;0t[1;34r(B[m[4l[?7h[34;1H[?1049l[23;0;0t
[?1l>================================= Messages =================================
```

Comment:
this crutft is still here. do you know why?

Reply:
I have no clue, each time I run the python script it will output it there, doesn't change each time I run either.

Reply:
ok i looked a little into it. I think its because of your shell config and cause @cindyli-13 wrote the `generate_can` script to use the `shell=True` option. just manually delete this diff for this file for now imo.

TBH, the script should be fixed not to use `shell=True` to fix this issue (also because shell=True should just almost always be avoided because of performance and potential security vulnerability reasons), but that sounds like it should probs be addressed seperately